dasharo-performance/cpu-temperature.robot

```diff
@@ -124,12 +124,19 @@ CPU Temperature Without Load (Ubuntu 22.04)
     Detect Or Install Package    lm-sensors
     Execute Command In Terminal    sensors-detect --auto
     ${timer}=    Convert To Integer    0
+    ${overheating_cnt}=    Convert To Integer    0
     FOR    ${i}    IN RANGE    (${TEMPERATURE_TEST_DURATION} / ${TEMPERATURE_TEST_MEASURE_INTERVAL}) + 1
         Log To Console    \n ----------------------------------------------------------------
         Log To Console    ${timer} min.
         ${temperature}=    Get CPU Temperature CURRENT
         Log To Console    Current temperature: ${temperature}°C
-        Should Be True    ${temperature} < ${MAX_CPU_TEMP}
+        IF    ${temperature} > ${MAX_CPU_TEMP}
```

Comment:
@pkubaj I wonder what's the real problem is, and if it really fixes this.

It feels to me that the whole measurement procedure is not perfect, as we measure CPU temperature, which can rapidly change, only in 1 minute intervals, and we take the current value (not e.g. the mean value from last minute).

Reply:
Yes, the measurement procedure is not perfect, but that's how the CPU temperature monitoring works, that is it checks the current temperature. Checking the mean value from last minute is one thing, but if we're checking the mean, why stop at 1 minute when the test lasts an hour? We could just read the temperature 3600 times and get the mean after one hour, which should be much simpler to implement (provided that Robot supports arrays with 3600 elements).